R/nullModelApproxSE.R

```diff
@@ -0,0 +1,217 @@
+## estimate the ratio (se.ratio) of the true SE(score) to the fast SE(score)
+## update the null model to contain this parameter
+## this needs to be run before using the approx.score.se option in assocTestSingle
+## this is based off of the SAIGE variance approximation
+
+
```

Comment:
this is the wrapper to: 
- fit the null model
- calculate the score and the approx at a set of variants
- update and return the small null model with the se.correction factor